router/middlewares/middlewareJson.go

```diff
@@ -48,6 +52,12 @@ func (controller *jsonControllerImpl) Wrap(handler JSONHandler) func(response ht
 			return
 		}
 		response.Header().Set("Content-Type", "application/json")
-		response.Write(payload)
+		_, err = response.Write(payload)
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.